cfme/tests/services/test_service_catalogs.py

```diff
@@ -204,10 +206,15 @@ def test_no_template_catalog_item(provider, provisioning, vm_name, dialog, catal
                   description="my catalog", display_in=True, catalog=catalog, dialog=dialog)
     catalog_item.create()
     flash.assert_message_match("'Catalog/Name' is required")
+    if not provider.exists:
+        try:
+            setup_provider(provider.key)
+        except FlashMessageException as e:
+            e.skip_and_log("Provider failed to set up")
```

Comment:
Because there is a bug in which after deleting provider everything goes haywire in catalogs .
https://bugzilla.redhat.com/show_bug.cgi?id=1210541.
Adding a provider is necessary for the finally part (catalog.delete)